.github/workflows/release-dev.yml

```diff
@@ -0,0 +1,42 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
@1nv0k32 isn't this inverted? The `release-dev.yml` us the tag branches and the `release.yml` uses branch main?

Reply:
It's the same behaviour as it was before. When you merge something into main, if there was no tag, you were considering it as a dev release, and if you were tagging main, you were considering it as a release. I've done the same, but in two separate files.